### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231027

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1081] [Announcements]
# 系统
1. 不论是否单网口设备都开启 Docker to LAN 的 NAT，以免切换到旁路由模式后 Docker 容器没网
2. 修复 Dockerman （Docker 的 LUCI 界面）设置内存等资源限制的数值转换 BUG #1074
3. 修复 Dockerman 的容器状态页面从未显示任何数据的 BUG
4. 新增常用 USB 无线网卡驱动：RTL8812AU/RTL8821AU, RTL8822BU/RTL8822CU，RTL8821CU，RTL8723DU #971 #987 #993 #1037

# 应用：
* QuickStart：
  1. 旁路由向导增加 NAT 开关，可解决某些无线热点联网问题 #598
  2. 修复 Safari 浏览器里自定义复选框组件（SwitchBox）未显示问题 #812

# 设备
* x86：
  1. i915-oot 驱动更新到上游最新版本
  2. 新增常用 SAS 卡驱动 mpt3sas，可支持 LSI MPT Fusion 3.0 and 2.0, 例如 SAS2008/SAS2308/SAS3008/SAS3108 等方案 #333 #1052
  3. igc 驱动增加支持 I226-K 和 I226-LMvP #1058
  4. bnx2x 驱动支持 2.5G 速率 #351 #924 #1036

**Comment by xpn282** [comment]
我有一个问题，我的机子是铭凡gk41，内置Wi-Fi。但是安装iStoreOS，并没有”无线局域网“的选项，请问我应该如何打开的Wi-Fi功能？

**Reply by jjm2473** [reply]
要看有没有驱动支持，执行`lspci -nnk`看看

**Reply by xpn282** [reply]
已经输入指令，帮我看看，我看不懂，接下来该怎么操作？
![IMG_4707](https://github.com/istoreos/istoreos/assets/53807602/1b325892-f6e6-47fa-93f4-6898c060086a)
![IMG_4706](https://github.com/istoreos/istoreos/assets/53807602/48546b5d-569c-4238-a2dc-686e2ee211cc)

**Reply by jjm2473** [reply]
有驱动啊，`01:00.0`这个设备是 QCA9377 无线网卡，驱动是 `ath10k_pci`。
执行`iw list`看看有没有输出。
还有，开个issue，把内核日志贴出来。

**Reply by xpn282** [reply]
执行iw lis后，没有任何反应，没什么文字出现

**Reply by jjm2473** [reply]
开issue，贴日志

**Comment by blankhang** [comment]
请问我把安装IstoreOS的盘1TB M2的剩余空间都给扩充了之后 现在想把docker从其他硬盘迁移回本机硬盘 现在不让移动
![image](https://github.com/istoreos/istoreos/assets/3981276/c5bc00c7-7f35-44ab-87b7-771566157f80)

![image](https://github.com/istoreos/istoreos/assets/3981276/f55432cd-63d5-4f2b-8481-03840a43f7cb)

![image](https://github.com/istoreos/istoreos/assets/3981276/d201fe30-9045-486b-96a6-eb78f05e3464)

如何解决？

**Reply by jjm2473** [reply]
其实就是先把docker停掉，然后把docker目录复制到其他地方，然后再修改docker的数据路径，然后再启动docker。

**Reply by jjm2473** [reply]
你是把overlay分区扩大了？下次升级固件可能会失败或者数据丢失。
正确的扩容方式是把硬盘后面分个区（首页就有这个功能），挂载到/mnt/data1之类的，然后把docker数据迁移过去。

**Comment by wood-kite** [comment]
最近入手了一台n100的机器，但是配备的是i266-v的网口，能使用，但是目前连华硕的AX82U的2.5G口，显示的速度和测出来的速度都是1G，想问一下是不是目前这款网卡还没支持到2.5G？后续什么时候能支持到呢？

**Reply by blankhang** [reply]
我的N100 i266-v 就是 跑的原生 iStoreOS 局域网复制能有250MB/S左右 应该是跑满的
![image](https://github.com/istoreos/istoreos/assets/3981276/89d1ee3f-4a4e-41ab-b368-f41671405a94)

**Reply by jjm2473** [reply]
换网线试试

**Comment by HLLOUIS** [comment]
支持的USB网卡还是用不了为什么呢？
